slowish/tests.py

```diff
@@ -164,6 +164,15 @@ def file_view_put(self, container_name, path):
         request.META['HTTP_X_AUTH_TOKEN'] = self.user.token
         return container(request, self.user.account.id, container_name, path)
 
+    def file_view_delete(self, container_name, path):
+        url = reverse('file',
```

Comment:
This is one of those cases where I'm glad that we don't have an automated mechanism enforcing our style guideline. I know that our standard style would put the first argument onto its own line, but it looks *so* odd to me to see:
```
    url = reverse(
        'file',
        kwargs={...})
```

There's something about that single 4-letter word, (7 characters total), using up a whole line of screen real-estate that looks really wrong to me. Especially when "url = reverse(" is already so short that nothing is being pushed excessively far to the right. So I'd be willing to bend the style rule here. But I'd be happy to hear other thoughts.